File: 30DaysOfCode/day02_operators/operators.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main() {
    double mealCost;
    int tip, tax;
    
    cin >> mealCost >> tip >> tax;
    
    double totalCost = round(mealCost + (0.01 * tip * mealCost) + (0.01 * tax * mealCost));
    
    cout << "The total meal cost is " << totalCost << " dollars." << endl;
    
    return 0;
}

File: 30DaysOfCode/day02_operators/operators.rb
def get_total_cost_of_meal()
    meal_cost = gets.to_f
    tip = gets.to_i
    tax = gets.to_i

    total_cost = (meal_cost + (0.01 * tip * meal_cost) + (0.01 * tax * meal_cost)).round.to_i
    
    return total_cost
end

print("The total meal cost is ", get_total_cost_of_meal(), " dollars.")
